insta-unfollower.py

```diff
@@ -236,7 +247,7 @@ def main():
             print('rebuilding following list...', end='', flush=True)
         else:
             print('building following list...', end='', flush=True)
-        following_list = get_following_list()
+        following_list = [x for x in get_following_list() if x['username'] not in get_allow_list()]
```

Comment:
I think it's better to add the test before or after the if testing the verified status of an account at [Line 273](https://github.com/tuxity/insta-unfollower/blob/master/insta-unfollower.py#L273), as it's still important to know who isn't following you, but skipping the unfollow action.